conbench/app/runs.py

```diff
@@ -44,23 +44,23 @@ def badge(self) -> str:
 #   (imported name has type "Type[conbench.app.runs.Run]", local name has
 #     type "Type[conbench.entities.run.Run]")  [assignment]
 class ViewRun(AppEndpoint, ContextMixin, RunMixin, TimeSeriesPlotMixin):
-    def page(self, benchmarks, contender_run, form, run_id):
```

Comment:
contender_run and run_id were (hopefully) redundant

"contender_run" doesn't make sense here, this is the view onto a single run.